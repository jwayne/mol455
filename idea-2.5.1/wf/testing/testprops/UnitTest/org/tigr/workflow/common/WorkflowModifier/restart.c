#include <stdio.h>
#include <stdlib.h>
#include <unistd.h>

int main(int argc, char* argv[]){
  if (argc != 2){
    printf("%s\n", "You must supply exactly one argument:  an amount of memory to allocate (in bytes).");
  }
  printf("%s%d%s\n", "Allocating ", atoi(argv[1]), "...");
  malloc(atoi(argv[1]));
  sleep(100);
};
