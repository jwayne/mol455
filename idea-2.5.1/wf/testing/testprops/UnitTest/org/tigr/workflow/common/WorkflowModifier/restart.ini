; Config file for testing simple workflow with a command to create a directory
 
[1.1.1.1.1]
name = command-I1
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = I1

[1.1.1.1.2]
name = distributed-command-I2
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = I2

[1.1.1.1.3]
name = command-I3
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = I3

[1.1.1.2.1]
name = command-J1
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = J1

[1.1.1.2.2]
name = distributed-command-J2
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = J2

[1.1.1.3.1]
name = command-K1
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = K1

[1.1.1.3.2]
name = distributed-command-K2
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = K2

[1.1.1.3.3]
name = command-K3
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = K3

[1.1.1.4.1]
name = distributed-command-L1
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = L1

[1.1.1.4.2]
name = command-L2
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = L2

[1.1.2.1]
name = command-O
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = O

[1.1.2.2.1]
name = command-P1
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = P1

[1.1.2.2.2]
name = distributed-command-P2
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = P2

[1.1.2.2.3]
name = command-P3
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = P3

[1.1.2.3.1]
name = distributed-command-Q1
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = Q1

[1.1.2.3.2]
name = command-Q2
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = Q2

[1.1.2.4.1]
name = command-R1
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = R1

[1.1.2.4.2]
name = distributed-command-R2
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = R2

[1.1.2.4.3]
name = command-R3
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record-long
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = R3

[1.1.2.5.1]
name = command-S1
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = S1

[1.1.2.5.2]
name = distributed-command-S2
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record-long
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = S2

[1.1.2.8]
name = distributed-command-V
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = V

[1.1.3.1]
name = distributed-command-W
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = W

[1.1.3.2]
name = command-X
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = X

[1.2]
name = distributed-command-E
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-record
param.stdout = restart-out.txt
param.stdoutappend = restart-out.txt
arg = E

[2]
name = command-C-wait
param.command = /home/aegan/bin/projects/workflow-2005-11-15/workflow/testing/testprops/UnitTest/org/tigr/workflow/common/WorkflowModifier/restart-wait
arg = C
