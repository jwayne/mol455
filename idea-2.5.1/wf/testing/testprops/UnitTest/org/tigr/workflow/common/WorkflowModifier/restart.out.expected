I1
I2
I3
J1
J2
K1
K2
K3
L1
L2
M1
M2
M3
N1
N2
O
P1
P2
P3
Q1
Q2
R1
R2
R3
S1
S2
T1
T2
T3
U1
U2
T1
T2
T3
K1
K2
K3
O
P1
P2
P3
Q1
Q2
R1
R2
R3
J1
J2
O
P1
P2
P3
Q1
Q2
R1
R2
R3
S1
S2
I1
I2
I3
O
P1
P2
P3
Q1
Q2
R1
R2
R3
S1
S2
T1
T2
T3
U1
U2
V
W
X
E
C
S1
S2
T1
T2
T3
U1
U2
V
W
X
C
K2
K3
O
P1
P2
P3
Q1
Q2
R1
R2
R3
S1
S2
T1
T2
T3
U1
U2
V
W
X
C
T1
T2
T3
U1
U2
V
W
X
C
R1
R2
R3
S1
S2
T1
T2
T3
U1
U2
V
W
X
C
L1
L2
O
P1
P2
P3
Q1
Q2
R1
R2
R3
S1
S2
T1
T2
T3
U1
U2
V
W
X
C
Q1
Q2
R1
R2
R3
S1
S2
T1
T2
T3
U1
U2
V
W
X
C
N1
N2
O
P1
P2
P3
Q1
Q2
R1
R2
R3
S1
S2
T1
T2
T3
U1
U2
V
W
X
C
T2
T3
U1
U2
V
W
X
C
P1
P2
P3
Q1
Q2
R1
R2
R3
S1
S2
T1
T2
T3
U1
U2
V
W
X
C
N2
O
P1
P2
P3
Q1
Q2
R1
R2
R3
S1
S2
T1
T2
T3
U1
U2
V
W
X
C
M1
M2
M3
O
P1
P2
P3
Q1
Q2
R1
R2
R3
S1
S2
T1
T2
T3
U1
U2
V
W
X
C
U1
U2
V
W
X
C
E
C
